packages/electron/e2e/fixtures/electron-fixtures.ts

```
@@ -11,7 +11,7 @@
  * Extended test with Electron fixtures
  */
 export const test = base.extend<ElectronFixtures>({
-  electronApp: async (_fixtures, use) => {
+  electronApp: async ({}, use) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)